google/cloud/forseti/services/inventory/base/gcp.py

```diff
@@ -667,6 +667,19 @@ def iter_networks(self, projectid):
         for network in self.compute.get_networks(projectid):
             yield network
 
+    @create_lazy('compute', _create_compute)
+    def iter_computesnapshots(self, projectid):
```

Comment:
To be consistent, can we just call this `iter_snapshots`?  As other methods here, do not incorporate the api naming, i.e. `compute`.

Reply:
Happy to change it, for what its worth i used compute disks as a reference for much of what I did here, and that calls the function `iter_computedisks`.

Reply:
Good point, opening an issue to us to make the naming consistent here.   :)

Reply:
https://github.com/GoogleCloudPlatform/forseti-security/issues/1901